setup.py

```diff
@@ -1,12 +1,44 @@
 #!/usr/bin/env python
 from setuptools import find_packages
 from distutils.core import setup
+import os
+import re
+
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(this_directory, 'README.md')) as f:
+    long_description = f.read()
 
 
 package_name = "dbt-synapse"
-package_version = "0.18.0.1"
+
+
+# get this from a separate file
+def _dbt_synapse_version():
+    _version_path = os.path.join(
+        this_directory, 'dbt', 'adapters', 'synapse', '__version__.py'
+    )
+    _version_pattern = r'''version\s*=\s*["'](.+)["']'''
+    with open(_version_path) as f:
+        match = re.search(_version_pattern, f.read().strip())
+        if match is None:
+            raise ValueError(f'invalid version at {_version_path}')
+        return match.group(1)
+
+
+package_version = _dbt_synapse_version()
```

Comment:
new convention for me. I imagine we also now need a `__version__.py` file?

Reply:
yup, I added [dbt/adapters/synapse/__version__.py ](https://github.com/dbt-msft/dbt-synapse/pull/32/files#diff-5c776e264b5117fc1c1744ef595f525d53b67166a9ee0dd76bec2b7fcf5bcf04). Theoretically, this enables dbt to report plugin versions:
```
$ dbt --version
installed version: 0.18.1
   latest version: 0.18.1

Up to date!

Plugins:
  - redshift: 0.18.1
  - postgres: 0.18.1
```

It doesn't seem to work currently here, though. I wonder if it's because `dbt-sqlserver` is missing `__version__.py`.